Update azure-stack-app-service-before-you-get-started.md

apwestgarth opened the pull request:
Fixed wording around update, furnace checked and now final formatting and editing to get queued for publishing,

azurecla commented:
Hi __@victorar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (victorar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

apwestgarth committed:
Update azure-stack-app-service-before-you-get-started.md

Fixed wording around update, furnace checked and now final formatting and editing to get queued for publishing,
